### PR TITLE
Use ed25519 from Go standard lib

### PR DESCRIPTION
**Pull request by cesarkuroiwa**
ed25519 was introduced into the standard lib with [Go 1.13](https://golang.org/doc/go1.13#crypto/ed25519), so I believe it would be desired to use it, thus removing some external deps

### CODE REVIEW

**Comment by cesarkuroiwa** [comment]
Not sure what this means or if I can do anything to fix it

File: go.mod [hunk 1]
```diff
@@ -3,7 +3,6 @@ module github.com/miekg/dns
 go 1.12
```
Comment (tmthrgd) [inline go.mod:3]: You should bump this to the version that introduced crypto/ed25519.

**Comment by tmthrgd** [comment]
I have no objections to this. I think go1.12 is well outside what we intend to support anyway. As for the CI failure, I think that’s unrelated, but I’ll have to see if I can figure it out.

File: go.mod (hunk 1 above)
Comment (cesarkuroiwa) [reply go.mod]: My bad, sorry about that. Thanks

**Review by miekg — APPROVED** [review]

**Pull request merged**